app/infrastructure/parsers/lms_parser.py

```
@@ -0,0 +1,301 @@
+                title = cell_text
+                attendance = (
+                    True
+                    if week_attendance is True
+                    else self._extract_flag(cell=cells[-2])
+                )
+
+            data[-1]["attendances"].append({"title": title, "attendance": attendance})
+
+        return data
+
+    def _extract_flag(self, cell: BeautifulSoup):
```

Ruff (0.12.2)

Missing return type annotation for private function `_extract_flag`

(ANN202)